design-system/react/src/components/Alert/styles.ts

```diff
@@ -6,13 +6,13 @@ import { createStyle } from '~/hooks';
 function createVariantStyle(key: string) {
   const color = `$${toCamelCase(`semantic-ghost-${key}-color`)}`;
   const iconColor = `$${toCamelCase(`semantic-ghost-${key}-icon`)}`;
-  const background = `$${toCamelCase(`semantic-solid-${key}-bg`)}`;
+  const background = `$${toCamelCase(`semantic-ghost-${key}-disabled-bg`)}`;
```

Comment:
Why we removed solid here?

Reply:
this solid was only to create the left border, which we don't have anymore.